modules/release-notes/snip-bug-fix-rhidp-2728.adoc

```diff
@@ -0,0 +1,9 @@
+[id="bug-fix-rhidp-2728"]
+= Handle PII logging in Application Logs
+
+Before this update, {product-short} included user IP addresses in the application logs. 
+
+With this update, {product-short} does not include user IP addresses in the application logs.
```

Comment:
*dies laughing at how this seems to be written for 5 year olds*.

"We used to do a thing. Now, we don't do that thing." Yes, obviously?